index.js

```diff
@@ -84,7 +99,42 @@ const defaults = {
 		mainnet: '0x4A5b9B4aD08616D11F3A402FF7cBEAcB732a76C6',
 		testnet: '0x6292aa9a6650ae14fbf974e5029f36f95a1848fd',
 	},
+	RENBTC_ERC20_ADDRESSES: {
+		mainnet: '0xEB4C2781e4ebA804CE9a9803C67d0893436bB27D',
+		kovan: '0x9B2fE385cEDea62D839E4dE89B0A23EF4eacC717',
+		rinkeby: '0xEDC0C23864B041607D624E2d9a67916B6cf40F7a',
+	},
 	INITIAL_ISSUANCE: w3utils.toWei(`${100e6}`),
+	CROSS_DOMAIN_MESSAGE_GAS_LIMIT: `${3e6}`,
+	COLLATERAL_MANAGER: {
+		SYNTHS: ['sUSD', 'sBTC', 'sETH'],
```

Comment:
Change all synths starts with `z` 
```suggestion
		SYNTHS: ['zUSD', 'zBTC', 'zBNB'],
```

Same as below.